conda_build_config.yaml

```diff
@@ -42,29 +42,39 @@ cython:
   - 0.29
 cython_blis:
   - 0.4.*
+dataclasses:
+  - 0.7
 dm_tree:
   - 0.1.*
 eigen:
   - 3.3.7
 ffmpeg:
   - 4.2
+flatbuffers:
+  - 1.12.*
 freetype:
   - 2
 fortran_compiler_version:
   - 7.2.*                      # [x86_64]
   - 7.3.*                      # [ppc64le]
+fsspec:
+  - 0.8.*
 future:
   - 0.18.*
 gast:
   - 0.3.3
+google_auth:
+  - 1.23.*
+google_auth_oauthlib:
+  - 0.4.*
 google_pasta:
-  - 0.1.8
+  - 0.2
 googleapis_common_protos:
   - 1.52.*
 graphsurgeon:
   - 0.4.1
 grpcio:
-  - 1.16.1
+  - 1.32.*
```

Comment:
@jayfurmanek - I think we should change the versions of grpcio and opt_einsum to whatever latest is available which are 1.31.* and 3.1.0 respectively. Otherwise, we won't be able to get TF and dependent environment builds. We can always update these to newer versions when they are available.